src/context/env-plan.js

```diff
@@ -13,13 +13,15 @@ module.exports = plan =>
       planVariables.addUsingFunction('env', ({ assertPresent, os }) => {
         assertPresent({ os });
         const DEFAULT_FOREST_URL = 'https://api.forestadmin.com';
-        const FOREST_URL = process.env.FOREST_URL || DEFAULT_FOREST_URL;
+        const FOREST_URL =
+          process.env.FOREST_URL || process.env.FOREST_SERVER_URL || DEFAULT_FOREST_URL;
         const FOREST_URL_IS_DEFAULT = FOREST_URL === DEFAULT_FOREST_URL;
         return {
           DATABASE_SCHEMA: process.env.DATABASE_SCHEMA,
           DATABASE_URL: process.env.DATABASE_URL,
           FOREST_ENV_SECRET: process.env.FOREST_ENV_SECRET,
           FOREST_URL,
+          FOREST_SERVER_URL: FOREST_URL,
```

Comment:
Looks like a duplicate ?

Reply:
Yep, it's to simply handle the two of them. They are both equal to either `FOREST_URL` or `FOREST_SERVER_URL`.
This allows to use the two of them, since for the agentv2 commands we use the second one. 
Plus it allows us to use either when working on the CLI.

(We definitely messed up by having different variables for different agents...)

Reply:
🔴